### PR TITLE
Feature/display all books krd

### LINKED ISSUE #7: Create Home Component / css file

### LINKED ISSUE #9: Create BookContainer Component / css file

### LINKED ISSUE #10: Create BookCard Component / css file

### LINKED ISSUE #18: Utils file

### PR DESCRIPTION
**Pull request by krdurrett**
## What Features are Being Added?
- Fixed api call in componentDidMount to display all books on main page.
- Created a Nav component.
- Added id to BookCard.

## What is Being Refactored?
- App.js and BookContainer to pass down id prop.
## Where should the reviewer start?
- App.js

## How should this be manually tested?
-Cypress

## What are the relevant tickets?
-n/a

## Any Other Notes/Questions/Problems?
-n/a

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -12,94 +13,24 @@ class App extends Component {
...
+      error: ''
     }
   }
 
   componentDidMount = () => {
     apiCalls.getAllBooks()
       .then(booksData => {
         const cleanedBooksData = booksData.map(book => cleanBookData(book));
-        this.setState({ books: cleanBookData})
+        this.setState({ books: cleanedBooksData})
```
Comment (Fordo29) [inline src/components/App.js:25]: Nice catch! tiny typo

Comment (cagallo) [inline src/components/App.js:25]: Good catch! Now we don't need dummy data, woop!

File: src/components/Nav.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (Fordo29) [inline src/components/Nav.js:1]: Yay start of Nav Bar!

File: src/components/App.js (hunk 1 above)
Comment (ectweitmann) [reply src/components/App.js:25]: Ditto! Nice catch Kayla!

**Review by cagallo — APPROVED** [review]
Looks great Kayla! Time for testing!

**Review by ectweitmann — APPROVED** [review]
Great work!

**Pull request merged**